docs/src/threading.rst

```diff
@@ -132,6 +132,23 @@ Threads
 .. c:function:: int uv_thread_join(uv_thread_t *tid)
 .. c:function:: int uv_thread_equal(const uv_thread_t* t1, const uv_thread_t* t2)
 
+.. c:function:: int uv_thread_setpriority(uv_thread_t tid, int priority)
+    If the function succeeds, the return value is 0.
+    If the function fails, the return value is non-zero.
+    Setting the scheduling priority of the thread specified by tid. It requires elevated
+    privilege to set specific priorities on some platforms.
+    The priority can be set to the following constants. UV_THREAD_PRIORITY_HIGHEST, 
+    UV_THREAD_PRIORITY_ABOVE_NORMAL, UV_THREAD_PRIORITY_NORMAL, 
+    UV_THREAD_PRIORITY_BELOW_NORMAL, UV_THREAD_PRIORITY_LOWEST.
+    For Linux, when schedule policy is SCHED_OTHER (default), it sets nice value internally.
```

Comment:
"it sets nice value internally" reads kind of awkward and is probably not very comprehensible to people without a UNIX background.  Maybe leave it out altogether:
```suggestion
```